ote/src/clj/ote/integration/import/gtfs.clj

```diff
@@ -1,17 +1,35 @@
 (ns ote.integration.import.gtfs
   "GTFS file import functionality."
-  (:require [ote.components.http :as http]
+  (:require [amazonica.aws.s3 :as s3]
+            [ote.components.http :as http]
             [com.stuartsierra.component :as component]
             [compojure.core :refer [routes GET]]
             [clj-http.client :as http-client]
             [ote.util.zip :refer [read-zip]]
             [ote.gtfs.spec :as gtfs-spec]
-            [ote.gtfs.parse :as gtfs-parse]))
+            [ote.gtfs.parse :as gtfs-parse]
+            [specql.core :as specql]
+            [ote.db.gtfs :as gtfs]
+            [clojure.java.io :as io]
+            [digest]
+            [specql.op :as op]
+            [taoensso.timbre :as log]))
 
 (defn load-zip-from-url [url]
   (with-open [in (:body (http-client/get url {:as :stream}))]
     (read-zip in)))
 
+(defn load-file-from-url [url last-import-date]
+  (let [query-headers (if (nil? last-import-date)
+                        {:as :byte-array}
+                        {:headers {"If-Modified-Since" last-import-date}
```

Comment:
Is the date in valid format? (`If-Modified-Since: <day-name>, <day> <month> <year> <hour>:<minute>:<second> GMT`)

Reply:
Yes. For some reason it is. We tested it